simulator/src/plotArea.js

```
@@ -47,7 +47,7 @@ function getCycleStartX(cycleNumber) {
 }
 function changeHeight(dir) {
     plotHeight += dir ? 5 : -5;
-    plotHeight = dir ? plotHeight = Math.min(sh(50), plotHeight) : plotHeight = Math.max(sh(20), plotHeight);
+    plotHeight = dir ? plotHeight = Math.min(sh(60), plotHeight) : plotHeight = Math.max(sh(20), plotHeight);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)